aleph/views/bookmarks_api.py

```diff
@@ -176,5 +176,6 @@ def migrate():
         index_elements=[Bookmark.role_id, Bookmark.entity_id],
     )
     db.session.execute(stmt)
+    db.session.commit()
```

Comment:
I accidentally removed this line in a previous commit or rebase I think. I was a little surprised the existing test didn’t catch this. When running this in development, if the session is not committed, no bookmarks are created (expected). However, in a test environment, asserting that bookmarks exist does not fail.

Pretty sure the cause is related to SQLAlchemy’s session handling, autocommit/transaction isolation config or something like that, but wasn’t able to isolate it yet.

Reply:
Hm great point and I see no immediate answer to this. Anything wrapped in a session should be running in a transaction so I would expect this to require a commit. But perhaps the the default isolation level is autocommit? 🤔